docs/security/trust/trust_sandbox.md

```diff
@@ -101,8 +101,6 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
         RUN chmod +x /usr/bin/docker \
```

Comment:
I also see that the dockerfile uses https://master.dockerproject.org/ for running Docker. Perhaps we should change that to https://get.docker.com/builds/Linux/x86_64/docker-latest ? Fine to leave that for another PR if you want

@jfrazelle wdyt?